command/server_test.go

```diff
@@ -330,3 +330,65 @@ func TestServer_DevTLS(t *testing.T) {
 	require.Equal(t, 0, retCode, output)
 	require.Contains(t, output, `tls: "enabled"`)
 }
+
+// TestConfigureDevTLS verifies the various logic paths that flow through the
+// configureDevTLS function.
+func TestConfigureDevTLS(t *testing.T) {
+	testcases := []struct {
+		ServerCommand   *ServerCommand
+		DeferFuncNotNil bool
+		ConfigNotNil    bool
+		TLSDisable      bool
+		CertPathEmpty   bool
+		ErrNotNil       bool
+	}{
+		// flagDev is false, nothing will be configured.
```

Comment:
I've seen testcases where this comment is done as a string, which I think can look a little nicer (and I think sometimes shows up in a fancy way in some IDEs?), like this:
```
testCases := map[string]struct {
		templateCount int
		exitAfterAuth bool
	}{
		"one": {
			templateCount: 1,
		},
		"one_with_exit": {
			templateCount: 1,
			exitAfterAuth: true,
		},
		"many": {
			templateCount: 15,
		},
		"many_with_exit": {
			templateCount: 13,
			exitAfterAuth: true,
		},
	}
```
This is totally optional and non-blocking, but I think it would be stronger if these were strings as opposed to comments in a similar style

Reply:
I like that a lot because it also opens up the door to including that description in the failed assertion messages.  🙇 ❤️